content/python/concepts/json-module/terms/json-dump/json-dump.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Please change folder name from `json-dump` --> `dump` and file name from `json-dump.md` to `dump.md`